example/lib/widgets/sliver_auto_start.dart

```diff
@@ -9,27 +11,42 @@ class SliverAutoStart extends StatefulWidget {
 }
 
 class _SliverAutoStartState extends State<SliverAutoStart> {
-  // in DriveKit SDK, autoStart defaults to false
-  bool _autoStart = false;
-
   @override
   Widget build(BuildContext context) {
-    return SliverToBoxAdapter(
-      child: Row(
-        children: [
-          const Text('Auto Start'),
-          const Spacer(),
-          Switch(
-            value: _autoStart,
-            onChanged: (value) {
-              DrivekitTripAnalysis.instance.activateAutoStart(value);
-              setState(() {
-                _autoStart = value;
-              });
-            },
-          ),
-        ],
-      ),
+    return MultiSliver(
+      children: [
+        const Text('Auto Start'),
+        const SliverGap(4),
+        ElevatedButton(
+          onPressed: () {
+            DrivekitTripAnalysis.instance.activateAutoStart(true);
+          },
+          child: const Text('Activate auto start'),
+        ),
+        const SliverGap(4),
+        ElevatedButton(
+          onPressed: () {
+            DrivekitTripAnalysis.instance.activateAutoStart(false);
+          },
+          child: const Text('Disable auto start'),
+        ),
+        const SliverGap(4),
+        ElevatedButton(
+          onPressed: () async {
+            final isAutoStartActivated =
+                await DrivekitTripAnalysis.instance.isAutoStartActivated();
+            if (context.mounted) {
+              ScaffoldMessenger.of(context).showSnackBar(
+                SnackBar(
+                  backgroundColor: Theme.of(context).primaryColor,
+                  content: Text('Auto start value: $isAutoStartActivated'),
+                ),
+              );
+            }
+          },
+          child: const Text('Check auto start value?'),
```

Comment:
Les textes me paraissent hétérogènes et je remplacerais par :
- "**Enable** auto start"
- "Disable auto start"
- "**Is auto start enabled?**"